string picker_id
---
sting robot_id
